test/codeu/chat/common/CstatusTest.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+public class CstatusTest {
```

Comment:
Awesome to see tests. Great start! You might also think about handling error cases: is it possible for a user to enter bad data? What happens if you request the access of a non-member? Try to think about corner cases, not just the [happy path](https://en.wikipedia.org/wiki/Happy_path).